docs/api/token.md

```diff
@@ -0,0 +1,19 @@
+# Token Management API
+
+- **GET** `/v1/namespaces/{namespace}/generatetoken`
+  - **Description**: Creates a new service account and gets a token from it.
+  - **Path Parameter**:
+    - `namespace` - The namespace of the service account to create token from.
+    - `serviceAccountName` - The name of the service account to create token from.
+  - **Query Params**:
+    ```json
+    {
+      "durationSeconds": int
+    }
+    ```
+  - **Response**: Token from the created service account.
+    ```json
+    {
+       "token": "string",
+    }
+    ```
```

Comment:
Shouldn't there be 2 routes here? One that creates a new `serviceAccount` and `token` and one that given a `serviceAccount` `namespacedName` returns its `token`?

Reply:
I think that one router is enough. We can check if service account exists, and if yes, create a token from it. If not, create a new service account and get a token from it.